src/swap/saga.ts

```diff
@@ -62,12 +60,11 @@ function calculateEstimatedUsdValue({
 
 interface TrackedTx {
   tx: TransactionRequest | undefined
-  txHash: Hash | undefined
   txReceipt: TransactionReceipt | undefined
 }
 
 function getTxReceiptAnalyticsProperties(
-  { tx, txHash, txReceipt }: TrackedTx,
+  { tx, txReceipt }: TrackedTx,
```

Comment:
The reason I used `txHash` instead `txReceipt.transactionHash` is in case we fail to get the receipt. Could we keep it this way?

Reply:
ah right, makes sense. i've reverted this change